Cabal/Distribution/PackageDescription/Parse.hs

```diff
@@ -161,6 +162,14 @@ storeXFieldsPD _ _ = Nothing
 -- ---------------------------------------------------------------------------
 -- The Library type
 
+subLibFieldDescrs :: [FieldDescr Library]
+subLibFieldDescrs =
+  [ simpleField "library"
+                        showToken             parseTokenQ
+                        (fromMaybe "subLibFieldDescrs" . libName)
```

Comment:
(I hope that we can change how `PackageDescription` is pretty-printed, as AFAICS there no way to read it in)